### ISSUE TITLE
[URGENT] Cannot access to SeaDronesSee Object Detection v1 Dataset

### ISSUE THREAD

**Issue by bgno1** [Ben93kie/SeaDronesSee#37]
Hello. 
Our team is currently unable to access the ODv1 dataset through the official link (https://macvi.org/dataset). 
The link to the Nextcloud page is broken. 
This issue is critically affecting our ongoing work, which is already halfway through, and we are in desperate need of this dataset to continue our research.

Could you please help us resolve this issue. Thank you very much.

**Comment by Ben93kie** [comment]
Sry, which dataset?

**Comment by Ben93kie** [comment]
For me, only Object Detection is not found, but we recommended to use the v2 version anyways. Let me know, I'll reply asap

**Comment by Ben93kie** [comment]
Sry, we recommended using the v2 version. Is it inevitable for you to use the v1 version?

**Comment by bgno1** [comment]
Dear Sir,

Thank you for your prompt response.
Unfortunately, we must use the OD v1 dataset.
A paper we submitted, which is currently under review, utilized the OD v1 dataset.
The editors have indicated that they are unable to access the dataset from the official link.
Therefore, we urgently request that you restore the official link to the OD v1 dataset.
Thank you very much for your assistance.

**Comment by Ben93kie** [comment]
give me a few minutes

**Comment by Ben93kie** [comment]
Do you need the test set metadata annotations? If not, it would be easy and it is being uploaded to here right now (20 minutes or so):

https://cloud.cs.uni-tuebingen.de/index.php/s/aJQPHLGnke68M52

**Comment by bgno1** [comment]
Thank you very much. We do not need the metadata. We greatly appreciate your prompt response and assistance!

**Comment by bgno1** [comment]
> Do you need the test set metadata annotations? If not, it would be easy and it is being uploaded to here right now (20 minutes or so):
> 
> https://cloud.cs.uni-tuebingen.de/index.php/s/aJQPHLGnke68M52

Thank you again. The images folder in the provided link is missing the training set images. We kindly request your assistance in resolving this issue.

-----------------
The issue has been resolved. The training set images are now available, likely due to upload network delays. Thank you again.

**Comment by Ben93kie** [comment]
Yes, was still uploading. Let me know if any issue persists. You can also let me know what paper you published (if it gets accepted). Good luck!
